### PR TITLE
Add 4.6 to channels in annotations.yaml

### PR DESCRIPTION
**Pull request by warrenvw**
See title.

### CODE REVIEW

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

**Comment by markusthoemmes** [comment]
Retest will not fix this I believe. The issue is this

```
time="2020-10-02T18:10:20Z" level=error msg="permissive mode disabled" bundles="[image-registry.openshift-image-registry.svc:5000/openshift-marketplace/serverless-bundle]" error="Error adding package error loading bundle into db: no default channel specified for serverless-operator"
Error: Error adding package error loading bundle into db: no default channel specified for serverless-operator
```

File: olm-catalog/serverless-operator/metadata/annotations.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 annotations:
     operators.operatorframework.io.bundle.channel.default.v1: "4.3"
```
Comment (markusthoemmes) [inline olm-catalog/serverless-operator/metadata/annotations.yaml:2]: I guess we'll have to update this to something sensible for this version? :thinking:

**Comment by markusthoemmes** [comment]
```
 time="2020-10-06T11:13:36Z" level=error msg="cannot find UID/GID for user nobody: No subuid ranges found for user \"nobody\" in /etc/subuid - check rootless mode in man pages." 
```

Seems like you haven't updated the image.

**Comment by markusthoemmes** [comment]
Checked out the PR and repushed the image as defined in the Dockerfile.

/retest

**Comment by markusthoemmes** [comment]
/test 4.5-upstream-e2e-aws-ocp-45

Instant infra failure.

File: hack/lib/catalogsource.bash [hunk 2]
```diff
@@ -75,11 +78,8 @@ spec:
...
-          /bin/opm registry add                         -d index.db --mode=replaces -b docker.io/markusthoemmes/serverless-bundle:1.7.2 && \
-          /bin/opm registry add                         -d index.db --mode=replaces -b registry.svc.ci.openshift.org/openshift/openshift-serverless-v1.8.0:serverless-bundle && \
-          /bin/opm registry add                         -d index.db --mode=replaces -b registry.svc.ci.openshift.org/openshift/openshift-serverless-v1.9.0:serverless-bundle && \
-          /bin/opm registry add --container-tool=podman -d index.db --mode=replaces -b image-registry.openshift-image-registry.svc:5000/$OLM_NAMESPACE/serverless-bundle && \
-          /bin/opm registry serve -d index.db -p 50051
+          /bin/opm registry add -d index.db --container-tool=podman --mode=replaces -b quay.io/openshift-knative/serverless-bundle:1.7.2,registry.svc.ci.openshift.org/openshift/openshift-serverless-v1.8.0:serverless-bundle,registry.svc.ci.openshift.org/openshift/openshift-serverless-v1.9.0:serverless-bundle,image-registry.openshift-image-registry.svc:5000/$OLM_NAMESPACE/serverless-bundle && \
+          /bin/opm registry serve -d database/index.db -p 50051
```
Comment (markusthoemmes) [inline hack/lib/catalogsource.bash:82]:
```suggestion
          /bin/opm registry serve -d index.db -p 50051
```

**Comment by markusthoemmes** [comment]
```
Error: open database/index.db: no such file or directory
```

Posted the needed change.

**Comment by rhuss** [comment]
CI is passing, that looks good to me :) Let's move forward ...

/lgtm
/approve

**Pull request merged**